content/blog/2025/oss-update.md

```diff
@@ -0,0 +1,63 @@
+---
+title: New Features for OddContrast
+sub:
+author: oddbird
+date: 2025-03-13
```

Comment:
Again think this might as well get the date of when we post it. Do we want these to go out on the same day?